Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "bevy_proto"
 version = "0.2.1"
```

Comment:
Might be good to bump the minor version with this change at least.

Reply:
No I'll definitely bump the major version haha, just prefer to do that last in case other PRs or changes need to be merged before bumping. Just keeps everything on the same page 🙂